src/System.CommandLine.Tests/Binding/BindingTestCase.cs

```diff
@@ -6,20 +6,32 @@ namespace System.CommandLine.Tests.Binding
     public class BindingTestCase
     {
         private readonly Action<object> _assertBoundValue;
-        
+
         private BindingTestCase(
-            string commandLine,
+            string commandLineToken,
             Type parameterType,
             Action<object> assertBoundValue,
             string variationName)
         {
             _assertBoundValue = assertBoundValue;
             VariationName = variationName;
-            CommandLine = commandLine;
+            CommandLineTokens = new[] { commandLineToken };
             ParameterType = parameterType;
         }
 
-        public string CommandLine { get; }
+        private BindingTestCase(
```

Comment:
Rather than duplicating the assignments, should the public ctor just invoke this one?

Reply:
I changed it so that one constructor calls the other. Neither is public, forcing the use of the factory method which is generic and allows for strongly-typed assertions.